lib/smart_udap_harmonization_test_kit/smart_udap_openid_connect_group.rb

```diff
@@ -12,21 +12,33 @@ class SMART_UDAP_OpenIDConnectGroup < SMARTAppLaunch::OpenIDConnectGroup # ruboc
 
     config(
       inputs: {
-        client_id: {
-          name: :udap_client_id,
-          title: 'UDAP Client ID'
+        smart_auth_info: {
+          title: 'UDAP Auth Info',
+          options: {
+            mode: 'access',
+            components: [
+              {
+                name: :auth_type,
+                default: 'asymmetric',
+                locked: true
+              },
+              {
+                name: :kid, # to be hidden
+                locked: true
+              },
+              {
+                name: :jwks, # to be hidden
+                locked: true
+              },
+              {
+                name: :requested_scopes # additional component to display
+              }
+            ]
+          }
```

Comment:
Not sure if this should be access mode or auth. The only field required for tests here are client_id, scopes, and issue_time. leaning more towards access mode because of the `issue_time`.

Reply:
We can't use AuthInfo for UDAP.